### ISSUE TITLE
Version information in release packages

### ISSUE THREAD

**Issue by myitcv** [google/protobuf#1150]
I just downloaded the [`protoc-3.0.0-beta-2-linux-x86_64.zip`](https://github.com/google/protobuf/releases/download/v3.0.0-beta-2/protoc-3.0.0-beta-2-linux-x86_64) build of [`v3.0.0-beta-2`](https://github.com/google/protobuf/releases/tag/v3.0.0-beta-2) (which is working great, thanks!). However the version information is limited to the following:

```
$ ./protoc --version
libprotoc 3.0.0
```

Is there any way that this can be corrected to include the fact that this is in fact `3.0.0-beta-2`? Otherwise, assuming there will be another beta version, it would be impossible to detect (in a CI log for example, without further infrastructure) what version of protocol buffers was used.

**Comment by myitcv** [comment]
Could I also ask that the Linux build (again, using `v3.0.0-beta-2` as an example) be zipped with the following directory structure:

```
bin/protoc
include/google/protobuf/timestamp.proto
include/google/protobuf/empty.proto
include/google/protobuf/source_context.proto
include/google/protobuf/duration.proto
include/google/protobuf/field_mask.proto
include/google/protobuf/api.proto
include/google/protobuf/descriptor.proto
include/google/protobuf/struct.proto
include/google/protobuf/wrappers.proto
include/google/protobuf/type.proto
include/google/protobuf/any.proto
include/google/protobuf/compiler/plugin.proto
```

As it is, in the `v3.0.0-beta-2` build we are missing the `bin` and `include` parent directories, which makes extracting this directly into `/usr/local` (or similar) a non-automatic step.

Thanks

**Comment by myitcv** [comment]
@xfxyjwf any chance this could be easily fixed as part of the next (`v3.0.0-beta-3`?) milestone?

**Comment by xfxyjwf** [comment]
The next beta release is v3.0.0-beta-4. I'll see if I have time to include a fix for this problem.
